src/view/renderer.js

```diff
@@ -304,10 +304,12 @@ export default class Renderer {
 			return false;
```

Comment:
If we agree to do https://github.com/ckeditor/ckeditor5-engine/pull/676/files#r87222594, then this won't be needed – that case will be handled inside `viewPositionToDom()`.